tests/Security/Http/EventListener/CheckTwoFactorCodeReuseListenerTest.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Scheb\TwoFactorBundle\Tests\Security\Http\EventListener;
+
+use PHPUnit\Framework\Attributes\Test;
+use PHPUnit\Framework\MockObject\MockObject;
+use Psr\Cache\CacheItemInterface;
+use Psr\Cache\CacheItemPoolInterface;
+use Psr\EventDispatcher\EventDispatcherInterface;
+use Scheb\TwoFactorBundle\Security\Http\EventListener\CheckTwoFactorCodeListener;
+use Scheb\TwoFactorBundle\Security\Http\EventListener\CheckTwoFactorCodeReuseListener;
+use Scheb\TwoFactorBundle\Security\TwoFactor\Event\TwoFactorCodeCheckEvent;
+use Scheb\TwoFactorBundle\Security\TwoFactor\Event\TwoFactorCodeReusedEvent;
+use Scheb\TwoFactorBundle\Tests\TestCase;
+use stdClass;
+use Symfony\Component\Cache\CacheItem;
+use Symfony\Component\Security\Core\User\UserInterface;
+use function sha1;
+
+/**
+ * @property CheckTwoFactorCodeListener $listener
+ */
+class CheckTwoFactorCodeReuseListenerTest extends TestCase
```

Comment:
There should be at least one test to make sure the cache is written, meaning the `save()` method is called with a CacheItem that has the expected values.

Reply:
Test is added